docs/guides/mysql/pitr/restic/same-db/archiver.md

```
@@ -0,0 +1,538 @@
+title: Restore in the Same Database Using Restic Driver
+menu:
+  docs_{{ .version }}:
+    identifier: restic-same-db
+    name: Restore in the Same Database
+    parent: pitr-restic-mysql
+    weight: 20
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)